src/Shared/AsyncWebView/AsyncWebView.Android.cs

```diff
@@ -27,7 +24,7 @@ private async Task ClearCacheAndCookies(CancellationToken ct)
 				_logger.LogDebug("Clearing cache and the cookies.");
 			}
 
-			var webView = GetChildren(_webView)
+			var webView = GetChildren(_webView2)
 				.OfType<WebView>()
```

Comment:
Shouldn't this be `WebView2`?

Reply:
it's from Android.Webkit.WebView, going back to _webView will remove the confusion